.cicd/generate-pipeline.sh

```diff
@@ -668,6 +669,34 @@ EOF
     timeout: ${TIMEOUT:-10}
     skip: ${SKIP_UBUNTU_20_04}${SKIP_PACKAGE_BUILDER}${SKIP_LINUX}
 
+  - label: ":oracle_linux: Oracle_Linux 8.3 - Package Builder"
+    command:
+      - "buildkite-agent artifact download build.tar.gz . --step ':oracle_linux: Oracle_Linux 8.3 - Build' && tar -xzf build.tar.gz"
+      - "./.cicd/package.sh"
+    env:
+      IMAGE_TAG: "oracle_linux-8.3-$PLATFORM_TYPE"
+      PLATFORM_TYPE: $PLATFORM_TYPE
+      OS: "oracle_linux-8.3" # OS and PKGTYPE required for lambdas
+      PKGTYPE: "rpm"
+    agents:
+      queue: "$BUILDKITE_TEST_AGENT_QUEUE"
+    key: "oracle8pb"
+    timeout: ${TIMEOUT:-10}
+    skip: ${SKIP_ORACLE_LINUX_8_3}${SKIP_PACKAGE_BUILDER}${SKIP_LINUX}
```

Comment:
We should be less specific with the version of Oracle Linux anywhere it is used. I know we specified the minor version for CentOS (at least in some places), but this was a mistake resulting from a lack of understanding about how upgrades on the RHEL-family operating systems work.

Unfortunately, minor versions are not as rigid in RHEL-family systems as they are in Debian-family systems. For example, look what happens when I call `yum update`:

<img width="467" alt="Screen Shot 2021-09-21 at 9 27 42 AM" src="https://user-images.githubusercontent.com/34947245/135326635-274280b3-a063-4dc0-9854-ba0fd13448b5.png">

You can see `yum` changes the version of the operating system. Therefore, we should stop specifying the minor version of RHEL-family operating systems altogether because this is misleading to our developers. They might believe an error exists on Oracle Linux 8.3 when, in fact, `yum update` was called and they are observing an error on Oracle Linux 8.4.